service/matching/user_data_manager_test.go

```diff
@@ -971,3 +979,11 @@ func TestUserData_CheckPropagation(t *testing.T) {
 		}
 	}, 100*time.Millisecond, 5*time.Millisecond, "CheckTaskQueueUserDataPropagation did not return fast enough")
 }
+
+func defaultTqmTestOpts(controller *gomock.Controller) *tqmTestOpts {
+	return &tqmTestOpts{
+		config:             defaultTestConfig(),
+		dbq:                defaultTqId(),
+		matchingClientMock: matchingservicemock.NewMockMatchingServiceClient(controller),
+	}
+}
```

Comment:
^ moved here since it's the only file left that uses it.

Reply:
hmm, I feel like there's way too many of these test-related structs floating around, we should (eventually) consolidate them

Reply:
Yeah, I tried my best consolidating what I encountered. If this was a test suite, we could just merge the struct into that. But I considered this file out of scope for this PR.